### PR TITLE
Site Improvements.

### PR DESCRIPTION
**Pull request by NicosHere**
This is my first time making a pull request on github. 

Not sure if I'm doing this right.

### CODE REVIEW

**Comment by NicosHere** [comment]
I am commenting here to needlessly ping Mykyta and others.

**Comment by NicosHere** [comment]
I am commenting here to needlessly ping Mykyta and others - he thinks he's super duper cool by deleting the github log on discord.

**Review by thatgoofydev — CHANGES_REQUESTED** [review]
Could you please be patient, 6h for someone to look into a PR in not a long time at all! We do have other responsibilities and would also like some sleep. If you continue spamming us, we will have to report you to GitHub.

Saying that, I like the fadein and content transition changes.
However the links (icons) and footer changes I don't love as much. If you could remove those changes, we I'm willing to merge this PR. Otherwise I would suggest waiting for @nkomarn's opinion. (Again don't spam!)

Also keep in mind that the website is currently just a placeholder and doesn't have high priority right know. At the moment we are more focused on the feature set of Loom than the website.

**Comment by NicosHere** [comment]
Hey, so I sent you a dm directly on discord, because I think you misunderstood my comment. I have no coding knowledge, so I was just playing around with the site and my comments were mostly jokes.

I hope you understand - there's no need to implement this into the site and any spam towards you was unintended.

However - I will make the changes.

**Comment by NicosHere** [comment]
I have made the requested changes, also added another animation - if you prefer it without I can remove.

File: index.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
     <meta name="description" content="Loom is a next-generation Minecraft server software with a modern and extensive plugin API. It aims to replace projects like Spigot and CraftBukkit.">
     <meta name="keywords" content="Minecraft, Spigot, server, plugins, api, open-source">
     <meta name="author" content="Loom Developers">
-    <meta name="og:image" content="https://loomdev.org/img/loom-background.png">    
-
+    <meta name="og:image" content="https://loomdev.org/img/loom-background.png">   
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-awesome@1.1.7/css/fork-awesome.min.css" integrity="sha256-gsmEoJAws/Kd3CjuOQzLie5Q3yshhvmo7YNtBG7aaEY=" crossorigin="anonymous">
```
Comment (thatgoofydev) [inline index.html:16]: Is this link statement still necessary?

**Review by thatgoofydev — COMMENTED** [review]
Looks good. I also like the new animations.

File: index.html (hunk 1 above)
Comment (NicosHere) [reply index.html:16]: Not anymore - I got rid of the icons.

**Pull request closed without merging**